config/locales/en.yml

```diff
@@ -5,6 +5,7 @@ en:
   generic:
     warning: Warning
     contents: Contents
+    back_to_top: Back to top
```

Comment:
I guess it's better to put this under  `navigation`. There is already a key called `back`  

```
  navigation:
    back: Back
```

Reply:
Done, thanks.